src/tests/test_demographics.py

```diff
@@ -91,11 +92,11 @@ def test_death_rate():
     group_size = N // 2 // len(ages_to_try)
     ages = sum(([age] * group_size for age in ages_to_try), []) * 2
     sexes = [SexType.Female] * (N // 2) + [SexType.Male] * (N // 2)
-    population_data = pd.DataFrame({
-        'date_of_death': [None] * N,
-        'age': ages,
-        'sex': sexes
-    })
+    pop = Population(size=N, start_date=date(1989, 1, 1))
```

Comment:
We may want to think about having a pytest fixture that creates a population with some characteristics. Not a question for here/now though!